frontend/packages/ceph-storage-plugin/src/utils/bucket-class.ts

```diff
@@ -0,0 +1,19 @@
+import { bucketClassNameRegex } from '../constants/bucket-class';
+
+const hasConsecutivePeriodsOrHyphens = (name: string): boolean => {
+  for (let i = 0; i < name.length - 1; i++) {
+    if (name.charAt(i) === '.') {
+      if (name.charAt(i + 1) === '.') return true;
+    }
+    if (name.charAt(i) === '-') {
+      if (name.charAt(i + 1) === '-') return true;
+    }
+  }
+  return false;
+};
```

Comment:
couldn't this be a simple regex? 
`/(\.\.)|(--)/`

Reply:
i am keeping this the same. I can't think of a good regex that works for multiple cases. I'll change it later on.

Reply:
@vbnrh I have suggested a regex pattern above. Please use it.

Reply:
this regex was not working initially. It needed the flags. Thanks :)